controller/controller.go

```diff
@@ -769,3 +778,30 @@ func generateISO(rootDir string, md *model.SystemInstall, options args.Args) err
 
 	return err
 }
+
+// writeCustomConfig creates a config file in the installer image that overrides the default
+func writeCustomConfig(chrootPath string, md *model.SystemInstall) error {
+	customPath := path.Join(chrootPath, conf.CustomConfigDir)
+
+	info, err := os.Stat(customPath)
+	if os.IsNotExist(err) {
```

Comment:
Why not use `utils.FileExists()` ? Also I don't think you need to check if it's directory since the environment is very controlled, we're pretty sure there won't be a directory in that path.